added molex slimstack 53748

Shackmeister opened the pull request:
Datasheet: https://www.molex.com/pdm_docs/sd/537480408_sd.pdf
Script PR: https://github.com/pointhi/kicad-footprint-generator/pull/378
![Skærmbillede fra 2019-06-13 13-58-17](https://user-images.githubusercontent.com/5279723/59430748-5f735f80-8de3-11e9-9abf-d310f31925fa.png)



---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [X] Provide a URL to a datasheet for the footprint(s) you are contributing
- [X] An example screenshot image is very helpful 
- [X] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [X] Check the output of the Travis automated check scripts - fix any errors as required
- [X] Give a reason behind any intentional library convention rule violation.

poeschlr commented:
Is there a reason for the fab outline not to include the "wings"?
Courtyard clearance in x direction is too close to the pads. (pads are wider than area taken up by wings)

![Screenshot from 2019-09-23 17-56-57](https://user-images.githubusercontent.com/18327350/65442113-e648c700-de2b-11e9-982f-3933824c3349.png)

Shackmeister commented:
Hi  @poeschlr 

I have fixed the courtyard issue, added the wings to the FAB and fixed the chamfer to lign up with the 3D model.

![Skærmbillede fra 2019-12-18 15-21-18](https://user-images.githubusercontent.com/5279723/71093940-68862480-21aa-11ea-8c22-369002af81b7.png)

poeschlr commented:
Thanks
